rocketpy/simulation/flight.py

```diff
@@ -1707,7 +1738,30 @@ def u_dot_generalized(self, t, u, post_processing=False):
         else:
             drag_coeff = self.rocket.power_off_drag.get_value_opt(free_stream_mach)
         R3 += -0.5 * rho * (free_stream_speed**2) * self.rocket.area * (drag_coeff)
-
+        for airbrakes in self.rocket.airbrakes:
```

Comment:
duplicating code is bad... 

Should we encapsulate this into another function?

Or maaybe deprecate one of the `u_dot` functions...